### PR TITLE
Initial etesync support

### PR DESCRIPTION
**Pull request by untitaker**
- [x] tests leave repo in dirty state

### CODE REVIEW

File: vdirsyncer/vobject.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ class Item(object):
     VCARD'''
 
     def __init__(self, raw):
-        assert isinstance(raw, str)
+        assert isinstance(raw, str), type(raw)
```
Comment (hobarrera) [inline vdirsyncer/vobject.py]: What exactly does `type(raw)` do here? I don't quite understand the syntax.

Comment (untitaker) [inline vdirsyncer/vobject.py]: `type(raw)` is the error message. It's not related but should be there IMO either way.

Comment (hobarrera) [inline vdirsyncer/vobject.py]: Oh, cool, didn't know the syntax.

**Comment by geier** [comment]
IIRC, type(raw) will used as the AssertionError's parameter should
isinstance be Falseish

Quoting Hugo Osvaldo Barrera (2017-03-30 20:21:40)

**Pull request merged**